pkg/microservice/aslan/core/system/service/registry.go

```diff
@@ -199,42 +215,278 @@ func ListAllRepos(log *zap.SugaredLogger) ([]*RepoInfo, error) {
 	return repoInfos, nil
 }
 
+type imageReqJob struct {
+	regService   registry.Service
+	endPoint     registry.Endpoint
+	registryInfo *models.RegistryNamespace
+	repoImages   *registry.Repo
+	tag          string
+	failedTimes  int
+}
+
+func goWithRecover(fn func(), logger *zap.SugaredLogger) {
+	go func() {
+		defer func() {
+			if r := recover(); r != nil {
+				logger.Panicf("Recovered from panic: %v", r)
+			}
```

Comment:
You can print the stack here. use `debug.PrintStack()`, `debug` is `"runtime/debug"`